vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -107,8 +107,8 @@ $global:imagesToPull += @(
     "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.30.2-windows-hp", # for k8s 1.29.x
     "mcr.microsoft.com/oss/kubernetes-csi/azurefile-csi:v1.30.3-windows-hp", # for k8s 1.29.x
     # Addon of Azure secrets store. Owner: jiashun0011 (Jiashun Liu)
-    "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.4.3",
-    "mcr.microsoft.com/oss/azure/secrets-store/provider-azure:v1.5.2",
+    "mcr.microsoft.com/oss/kubernetes-csi/secrets-store/driver:v1.4.4",
```

Comment:
Try to keep the old one if it is being used in production now. same for provider-azure

Reply:
sounds good